pkg/cimfs/cimfs.go

```diff
@@ -13,5 +15,61 @@ func IsCimFSSupported() bool {
 	if err != nil {
 		logrus.WithError(err).Warn("get build revision")
 	}
-	return osversion.Build() == 20348 && rv >= 2031
+	// TODO(ambarve): add proper version for post iron builds
+	return true || (osversion.Build() == 20348 && rv >= 2031)
+}
+
+// IsBlockedCimSupported returns true if block formatted CIMs (i.e block device CIM &
+// single file CIM) are supported on the current OS build.
+func IsBlockedCimSupported() bool {
```

Comment:
Can we be consistent about if `cim` is written as `Cim` or `CIM`?

Reply:
I think we will have to modify some existing code for this too. I would prefer to do this in a separate PR to avoid making these commits larger. Is that okay?

Reply:
Okay, for now just be consistent for new code please